app/models/product.rb

```diff
@@ -6,8 +6,12 @@ class Product < ApplicationRecord
   has_many :favourites
   has_many :line_items
   has_many :orders, through: :line_items
-  has_many :reviews, through: :orders, dependent: :destroy
```

Comment:
can check with ZH on this pls and save the new schema in the kitt diagram so everyone can see the new schema